README.md

```diff
@@ -68,24 +68,24 @@ in Sass):
 #### `asset-path($relative-asset-path)`
 Returns a string to the asset.
 
-* `asset-path("rails.png")` becomes `"/assets/rails.png"`
```

Comment:
Maybe `returns` would be better for this case since we are talking about the method?